kolibri/plugins/html5_app_renderer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -69,6 +71,31 @@
       }
       this.$emit('stopTracking');
     },
+    methods: {
+      iframeMessageReceived(event) {
+        if (!event) {
+          return;
+        }
+        var message = JSON.parse(event.data);
+        if (message.action === 'stateUpdated') {
```

Comment:
Might want to add these constants somewhere so we are not directly comparing strings.

Reply:
I would save in a constants file inside the hashi package. Then we can reference them from here using `import { MESSAGES } from 'hashi/src/constants';`.